src/DynamoUtilities/Md2Html.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Dynamo.Utilities
+{
+    public class Md2Html
```

Comment:
do we expect others to use this utilty? If not maybe we make it internal?

Reply:
Yes, internal makes more sense.